_updatePublisher.sh

```diff
@@ -32,7 +32,7 @@ done
 
 echo "Checking internet connection"
 case "$OSTYPE" in
-	linux-gnu* ) wget -q --spider http://tx.fhir.org/snomed/doco ;; 
+	linux-gnu* ) ping tx.fhir.org -4 -c 1 -w 1000 >/dev/null ;;
```

Comment:
This change comes from allowing the script to auto update.

Reply:
Laz said there was something odd about `ping` - it may have only been on our other repo

Reply:
Yeah let me revert it as the updated `ping` command doesn't seem to work in github actions.